File: client-web/src/app/core/models/notification.models.ts
export enum NotificationType {
  NewOrder = 0,
  OrderUpdated = 1,
  NewCustomerRegistered = 2
}

export interface AppNotification {
  id: string;
  type: NotificationType;
  title: string;
  message: string;
  entityId?: string;
  isRead: boolean;
  createdAtUtc: string;
}

File: LaundryMgmt.Mobile/Models/NotificationModels.cs
namespace LaundryMgmt.Mobile.Models;

public enum NotificationType
{
    NewOrder = 0,
    OrderUpdated = 1,
    NewCustomerRegistered = 2,
    PromotionCreated = 3
}

public record NotificationDto(
    Guid Id, NotificationType Type, string Title, string Message, string? EntityId, bool IsRead, DateTimeOffset CreatedAtUtc);
